engine/class-methods-without-function-keyword.yay

```diff
@@ -0,0 +1,14 @@
+$(macro: recursion) {
+    $(\Pre\Standard\Parser\visibilityModifiers())
+    $(T_STRING as name)
+    (
+    $(optional(\Pre\Standard\Parser\arguments()))
+    )
+    $(optional(chain(token(':'), \Pre\Standard\Parser\type())) as type)
```

Comment:
I added that missing `\Pre\Standard\Parser\returnType()` last night, so that should make this a bit simpler...